index.html

```diff
@@ -40,6 +42,30 @@ <h3>I’am a Software Developer</h3>
                 <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
             </ul>
         </section>
```

Comment:
- [ ] As the first reviewer requested To make your webpage more semantically correct kindly wrap the headline section in a `<section>` tag. 
- [ ] Kindly align the title, subtitle, and the paragraph to the left just like the original design.
![image](https://user-images.githubusercontent.com/43698511/133664752-60d59505-af10-4cec-aa15-aa8098698e37.png)